shared/getting-started/deployingCode/nodejs.md

```diff
@@ -1,5 +1,6 @@
-A nice first project to get started is a simple [Express.js][expressjs] web server which, will serve a static page on port `:80`. All the project source code can be found [here on GitHub][simple-server-node-link]. Download a zipped file of the project [here][github-download], and when downloaded, unzip the file and open a terminal in the root of the project directory.
+A nice first project to get started is a simple [Express.js][expressjs] web server which, will serve a static page on port `:80`. All the project source code can be found [here on GitHub][simple-server-node-link]. Download a zipped file of the project [here][github-download-js], and when downloaded, unzip the file and open a terminal in the root of the project directory. For a TypeScript version of the project please go [here][github-download-ts].
```

Comment:
Nitpick but how about?

```suggestion
A nice first project to get started is a simple [Express.js][expressjs] web server which, will serve a static page on port `:80`. All the project source code can be found [here on GitHub][simple-server-node-link]. Download a zipped file of the project [here][github-download-js], and when downloaded, unzip the file and open a terminal in the root of the project directory. A similar TypeScript version of this project can be found [here on GitHub][github-download-ts].
```

Reply:
I will also bring @pdcastro for a quick look at the wording. You do have a way with things, man.

Reply:
Pre-existing, but: `which, will serve` → `which will serve`
Or even better: `which, will serve` → `that serves`

I understand that the best practice for links is that they should be the name of the thing being linked, rather than the word "here" or "here on GitHub". Thus:

`All the project source code can be found [here on GitHub][simple-server-node-link].` →
`All the project source code can be found on GitHub: [balena-node-hello-world][balena-node-hello-world].`

Also rewording `simple-server-node` to `balena-node-hello-world` because the repo itself was renamed -- see other PR comment.

Actually, I think the paragraph overall would read better as follows:

> A nice first project to get started is `balena-node-hello-world`, an `Express.js` web server that serves a static page on port `:80`. Download a zipped file of the project, `master.zip (JavaScript)` or `master.zip (TypeScript)`, unzip it and open a terminal in the root of the project directory.

The markdown being:
```
A nice first project to get started is [balena-node-hello-world][balena-node-hello-world],
an [Express.js][expressjs] web server that serves a static page on port `:80`.
Download a zipped file of the project, [master.zip (JavaScript)][github-download-js]
or [master.zip (TypeScript)][github-download-ts], unzip it and open a terminal in the
root of the project directory.
```

Reply:
I agree with links named as to what they are pointing to, rather than `here`. The TS project being a more modern repo, has `main` as the main branch, so updated the text and link accordingly.